docs/getting-started/1-concepts.md

```diff
@@ -4,18 +4,44 @@ The power of integrating linting into the design-first workflow, or any workflow
 
 To achieve this, Spectral has three key concepts:
 
-- **Rulesets** act as a container for rules and functions.
+- **Rulesets** act as a container for rules and functions. 
 - **Rules** filter your object down to a set of target values and specify the function that is used to evaluate those values.
 - **Functions** accept a value and return issues if the value is incorrect.
 
 Rules can be comprised of one of more functions. For example:
 
-- HTTP Basic is not allowed at this company
+- HTTP Basic is not allowed
 - All operations are secured with a security schema
 - Descriptions must not contain Markdown
 - Tags must be plural
 - Tags must be singular
 
-Spectral comes bundled with a [set of core functions](../reference/functions.md) and default style guides for [OpenAPI v2 and v3](./4-openapi.md) and [AsyncAPI v2](./5-asyncapi.md), which you can extend, cherry-pick, or disable entirely.
+## Core Rulesets
+
+Spectral is a generic linter, but comes bundled with a [set of core functions](../reference/functions.md) and default style guides for OpenAPI v2.x and v3.x and AsyncAPI v2. You can extend, cherry-pick, or disable rules to meet your needs. 
```

Comment:
Technically, core functions are no longer bundled with the core of Spectral itself.
We do, however, still officially maintain & support them - they just live in a separate package.
Having said that, for users it doesn't probably matter, thus we can keep it as is I suppose.